frontend/src/lib/environment-provider.tsx

```diff
@@ -30,7 +29,7 @@ export const EnvironmentProvider: FCWC = ({ children }) => {
         return <LoadingAnimation />
     }
 
-    if (!env.user.userId && (ENV.IS_PROD_MODE || !location.pathname.startsWith('/dev/user'))) {
+    if (!env.user.userId && (ENV.IS_PROD_MODE || !window.location.pathname.startsWith('/dev/user'))) {
```

Comment:
this blue up because useLocation wasn't being called inside a route provider.  I guess the old version of the router was more forgiving